rpc/js/vite.config.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference types="vitest" />
+import path from 'node:path';
+import { defineConfig } from 'vite';
+
+import pkg from './package.json';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  define: {
+    'process.env.NODE_ENV': '"production"',
+    __VERSION__: JSON.stringify(pkg.version),
+  },
+  build: {
+    commonjsOptions: {
+      transformMixedEsModules: true,
+      include: [
+        /exponential-backoff/u,
+        /google-protobuf/u,
+        /@improbable-eng\/grpc-web/u,
+        /gen\//u,
+      ],
+    },
+    minify: true,
```

Comment:
I don't think this option is required; `vite` will [minify by default](https://vitejs.dev/config/build-options.html#build-minify)